linux/system/repo.sls

```diff
@@ -132,6 +140,9 @@ linux_repo_{{ name }}:
   {%- if repo.key_server is defined %}
   - keyserver: {{ repo.key_server }}
   {%- endif %}
+  {%- if repo.key_url is defined and grains['saltversioninfo'] >= [2017, 7] %}
```

Comment:
`or repo.key_url.startswith('salt://')`

Reply:
I mean:
`{%- if repo.key_url is defined and (grains['saltversioninfo'] >= [2017, 7] or repo.key_url.startswith('salt://')) %}`

Reply:
well, you are right, can you send one more PR (I will merge it on GH while traveling home)

Reply:
See PR #156